setup/providers/azure.md

```diff
@@ -7,6 +7,8 @@ sidebar:
 
 {% include toc %}
 
+If you want to configure Spinnaker to deploy containerized apps to Kubernetes on Azure, follow the steps [here](http://aka.ms/azspinlab). Otherwise, if you would like to deploy to Azure VM Scale Sets, follow the steps below.
```

Comment:
I would link to the Kubernetes setup page before I linked to the codelab. According the main setup page, the setup sections are for "a more serious, scalable deployment of Spinnaker", which is not the codelab. I'm also fine if you link to both.  Here's what I suggest:

> This page describes how to setup Spinnaker to target VM Scale Sets in Azure. If you want Spinnaker to deploy containerized apps to Kubernetes on Azure instead, follow the steps [here](/setup/providers/kubernetes) or try the codelab [here](/guides/tutorials/codelabs/azure-kubernetes-source-to-prod).